Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -1535,6 +1554,12 @@ void Scene_c3t3_item::show_intersection(bool b)
 void Scene_c3t3_item::show_cnc(bool b)
 {
   d->cnc_are_shown = b;
+  Q_FOREACH(QAction* action, contextMenu()->actions())
+    if(action->objectName() == "actionShowCNC")
+    {
+      action->setChecked(b);
+      break;
+    }
```

Comment:
Use `findChild`.

And anyway, as soon as I use the same code pattern more than twice, I create a function. You should do the same.